python-checks/src/test/resources/checks/itemOperationsTypeCheck/itemOperations_delitem.py

```diff
@@ -21,7 +21,7 @@ def builtin_types_not_supporting_delitem():
   del mydict.items()[0]  # Noncompliant
 
   # iterators
-  del iter(mylist)[0]  # Noncompliant
+  del iter(mylist)[0]  # FN Noncompliant
```

Comment:
From what I see, `iter` was always ambiguous. It used to have 3 possible definitions that all returned an `Iterator`. Now there is a new definition that returns a `_SupportsNextT` as well.